### ISSUE TITLE
Unable to access my account crypto defi 

### ISSUE THREAD

**Issue by 091470** [crypto-com/chain-desktop-wallet#1423]
I see alot of folks have been  having this issue. I can see my address on web 3 that's all how can regain control of my account address. I make a deposit its transferred before I can manage it.

**Comment by crypto-matto** [comment]
Hello @091470, can you briefly describe the issue you're facing? Provided with your flow, with some screenshots? Thanks.

**Comment by 091470** [comment]
Yes thank you i have been trying to access my account i have verified signature . And still haven't gained any access. I am sending a screenshot of one of the addresses 433 I could really use some intel. On what i should be doing

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
